### ISSUE TITLE
Filter Selection is not accessible

### ISSUE THREAD

**Issue by Cybermage83** [Kuechlin/mantine-data-grid#123]
## Description
When tabbing into filtered dropped down for selection of option via cursor keys, there is no accessibility values

No accessibility  value on option, even though it is present in meantime core select
## Expected Behavior
When going up or down with cursor keys over the selection there should be an accessibility value, but there is none

Value of the option should be accessible
